decidim-core/app/queries/decidim/own_activities.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Decidim
+  class OwnActivities < PublicActivities
+    def query
```

Comment:
Since this is otherwise exactly the same as in the parent class, I suggest that we extract the visibility argument to its own method and just override that in the `OwnActivities` class.

Reply:
Refactored